charts/backstage/Chart.yaml

```
@@ -12,7 +12,11 @@ annotations:
   charts.openshift.io/provider: Red Hat Developer Hub Team
   charts.openshift.io/supportURL: https://github.com/redhat-developer/rhdh-chart/issues
 apiVersion: v2
-description: A Helm chart for deploying RHDH (a Backstage application)
+description: |
+  A Helm chart for deploying RHDH (a Backstage application)
```

I wonder if this might be simpler/better. 

```suggestion
  A Helm chart for deploying Red Hat Developer Hub
```